README.md

```diff
@@ -1,75 +1,132 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
 
-> Description the project.
+# FINAL CAPSTONE (BACK END)
```

Comment:
- [ ] You are using Rspec, but the folder "test" still exists, which is used for another test framework.  Please, consider deleting that folder if you are not using it 💪